src/theory/quantifiers/extended_rewrite.cpp

```diff
@@ -1563,6 +1585,571 @@ Node ExtendedRewriter::extendedRewriteArith(Node ret)
   return new_ret;
 }
 
+int ExtendedRewriter::bitVectorSubsume(Node a, Node b, bool strict, bool tryNot)
+{
+  Assert(a.getType() == b.getType());
+  int max_ret = strict ? 2 : 1;
+  int curr_ret = 0;
+  Trace("q-ext-rewrite-debug2")
+      << "Subsume " << a << " " << b << "?" << std::endl;
+  // simple cases : a=b, a=max, b=0
+  if (a == b || isConstBv(a, true) || isConstBv(b, false))
```

Comment:
`isConstBv` only tests for zero and ones (depending on flag `isNot`), IMO its name is quite misleading.
Also, it is actually not really necessary, you could use `bv::utils::isZero()` to test for zero (https://github.com/CVC4/CVC4/blob/master/src/theory/bv/theory_bv_utils.h#L56) and add a corresponding `isOnes()`. That would be way cleaner and `isConstBv` would then be obsolete.